src/olympia/devhub/templates/devhub/addons/listing/item_actions.html

```diff
@@ -47,12 +47,20 @@ <h5>{{ _('Actions') }}</h5>
       </li>
     {% endif %}
   {% endif %}
-  {% if not addon.disabled_by_user and show_listed and addon.status == amo.STATUS_APPROVED %}
-    <li>
-      <a href="{{ url('stats.overview', addon.slug) }}" class="tooltip"
-         title="{{ _('Daily statistics on downloads and users') }}">
-        {{ _('Statistics') }}</a>
-    </li>
+  {% if not addon.disabled_by_user and addon.status == amo.STATUS_APPROVED %}
+    {% if waffle.flag('beta-stats') %}
```

Comment:
Are we actually planning to segment the people who have access by user or enable it for a % of users?  Not sure why we're not just using waffle.switch.

Reply:
% of users